src/animations/timeLines.js

```diff
@@ -13,19 +13,20 @@ import {
 
 gsap.registerPlugin(MotionPathPlugin);
 
-export const moveHideItemAndChangeContentTl = (
+export const hideItemAndChangeContentTl = (
   itemRef,
   changeContent,
   newContentId
 ) => {
   const tl = gsap.timeline({ defaults: { ease: 'power4.out' } });
-  return tl
-    .to(itemRef, {
-      y: '+=220',
-      duration: 1,
-    })
-    .to(itemRef, { autoAlpha: 0, delay: 1 })
-    .eventCallback('onComplete', changeContent, [newContentId]);
+
+  return tl.to(itemRef, {
+    autoAlpha: 0,
+    duration: 1,
+    delay: 1,
+    onComplete: changeContent,
+    onCompleteParams: [newContentId],
```

Comment:
you could use more javascript instead of gsap here:
`onComplete: () => changeContent(newContentId)`